src/types/index.ts

```diff
@@ -55,8 +55,18 @@ export interface TransferFinishedErrorPayload {
   errorMessage: string;
 }
 
+export interface IntegrationAccessToken {
+  accountId: string
+  accountName: string
+  accessToken: string
+  brokerType: string
+  brokerName: string
+}
+
 export interface LinkOptions {
```

Comment:
on iOS SDK we have the following

> public class LinkConfiguration {
>     var linkToken: String
>     var settings: LinkSettings? ...

> public struct LinkSettings {
>     public var useSecureOnDeviceStorage: Bool
>     public var accessTokens: [IntegrationAccessToken]?
>     public var transferDestinationTokens: [IntegrationAccessToken]?

can we make it consisten across all SDKs ?

Reply:
I've harmonized the naming of settings and extra props.